tests/end2end/features/caret.feature

```diff
@@ -323,6 +323,8 @@ Feature: Caret mode
 
     # Search + caret mode
 
+    # https://bugreports.qt.io/browse/QTBUG-60673
+    @qt!=5.8.0 @qt!=5.9.0 @qt!=5.9.1 @qt!=5.9.2 @qt!=5.9.3 @qt!=5.9.4
```

Comment:
This is the only way I could think of doing this without adding a specific decorator for this case. 

Would you prefer adding a `@no_qt_focus_bug` or similar?